google/auth/pluggable.py

```diff
@@ -132,7 +132,7 @@ def __init__(
         self._credential_source_executable_output_file = self._credential_source_executable.get(
             "output_file"
         )
-        self._tokeninfo_username = kwargs.get("tokeninfo_username", "")  # dummy value
+        self._tokeninfo_username = kwargs.get("_tokeninfo_username", "")  # dummy value
```

Comment:
Why was this not caught earlier? Is a unit test missing?

Reply:
Unfortunately there wasn't a case to test that and didn't break the coverage rate.

Reply:
Can you add a test case to catch this?

Reply:
Test added for variable injection. Also I realized it's not a good idea to put it into constructor.